File: pgm/ABC_R/ABC_412_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
blank <- ""
space <- " "
yn <- "\n"
one <- 1L
two <- 2L
zero <- 0L
N <- arr[one] |>
  as.integer()
cnt <- zero
for (i in seq_len(N)) {
  A <- strsplit(x = arr[i + one], split = space, fixed = TRUE)[[one]][one] |>
    as.integer()
  B <- strsplit(x = arr[i + one], split = space, fixed = TRUE)[[one]][two] |>
    as.integer()
  if (A < B) {
    cnt <- cnt + one
  }
}
cnt |>
  cat(yn, sep = blank)
q("no")

File: pgm/ABC_PHP/ABC_412_A.php
<?php
$arr = file('php://stdin', FILE_IGNORE_NEW_LINES);
const space = ' ';
const one   = 1;
const zero  = 0;
$N = (int) $arr[zero];
$cnt = zero;
for ($i = one; $i <= $N; $i++) {
    [$A, $B] = array_map('intval', explode(space, $arr[$i]));
    if ($A < $B) {
        $cnt++;
    }
}
echo $cnt . PHP_EOL;
exit(0);
